src/SIL.XForge.Scripture/ClientApp/.storybook/preview-head.html

```diff
@@ -1,2 +1,9 @@
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" />
 <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons|Material+Icons+Outlined" />
+
+<script>
+  // Allows global font effects from mdc typography
+  setTimeout(() => {
+    document.body.classList.add("mdc-typography");
```

Comment:
Did you look into whether it might be possible to do this using a decorator? (A Storybook decorator, not a TS decorator). Does it need to be on the body element, or just an ancestor of the component under test?